lib/tasks/rubocop.rake

```diff
@@ -13,6 +15,20 @@
 #
 #++
 
+def rubocop_bin
+  return @rubocop_bin if @rubocop_bin
+  return @rubocop_bin = ENV["RUBOCOP_BIN"] if ENV["RUBOCOP_BIN"]
+
+  version = system("grep 'rubocop-0.71.0' .rubocop.yml") ? "0.71.0" : "0.41.2"
```

Comment:
Um, I do not like much running external commands for simple things which can be done in native Ruby...
And BTW we need to be careful about the dots in the grep command, they need to be escaped otherwise it would match also a text like `rubocop-037120`.

What about using `File.read(".rubocop.yml").include?("rubocop-0.71.0")` instead?

```suggestion
  version = File.read(".rubocop.yml").include?("rubocop-0.71.0") ? "0.71.0" : "0.41.2"
```